test/gurka/test_instuctions_roundabout.cc

```diff
@@ -0,0 +1,175 @@
+#include "gurka.h"
+#include <gtest/gtest.h>
+
+#if !defined(VALHALLA_SOURCE_DIR)
+#define VALHALLA_SOURCE_DIR
+#endif
+
+using namespace valhalla;
+
+class InstructionsRoundabout : public ::testing::Test {
+protected:
+  static gurka::map map;
+
+  static void SetUpTestSuite() {
+    constexpr double gridsize = 100;
+
+    const std::string ascii_map = R"(
+            I
+            |
+            H
+           / \
+       A--B   E--F-G
+           \ /
+            C
+            |
+            D
+    )";
+
+    const gurka::ways ways =
+        {{"AB",
+          {{"highway", "primary"},
+           {"name", "East Governor Road"},
+           {"ref", "A 1"},
+           {"direction", "East"},
+           {"destination", "München;Kürten;Dhünn"},
+           {"destination:ref", "A 95;B 2;B 300"}}},
+         {"BCEHB",
+          {{"highway", "primary"}, {"oneway", "yes"}, {"junction", "roundabout"}, {"name", ""}}},
+         {"CD", {{"highway", "primary"}, {"name", "Homestead Lane"}}},
+         {"EF",
+          {{"highway", "primary"},
+           {"name", "East Governor Road"},
+           {"ref", "A 1"},
+           {"direction", "East"}}},
+         {"FG", {{"highway", "primary"}, {"ref", "A 1"}, {"direction", "East"}}},
+         {"HI", {{"highway", "primary"}, {"name", ""}}}};
+
+    const auto layout = gurka::detail::map_to_coordinates(ascii_map, 100, {5.1079374, 52.0887174});
+    map = gurka::buildtiles(layout, ways, {}, {}, "test/data/gurka_roundabouts",
+                            {{"mjolnir.data_processing.use_direction_on_ways", "true"},
+                             {"mjolnir.admin",
+                              {VALHALLA_SOURCE_DIR "test/data/netherlands_admin.sqlite"}}});
+  }
+};
+
+gurka::map InstructionsRoundabout::map = {};
+
+/*************************************************************/
+
+// enter_roundabout_verbal
+// "0": "Enter the roundabout."
+TEST_F(InstructionsRoundabout, RoundaboutEnterOnly) {
+  auto result = gurka::route(map, "A", "E", "auto");
+  gurka::assert::raw::expect_maneuvers(result, {DirectionsLeg_Maneuver_Type_kStart,
+                                                DirectionsLeg_Maneuver_Type_kRoundaboutEnter,
+                                                DirectionsLeg_Maneuver_Type_kDestination});
+  int maneuver_index = 1;
+
+  // TODO: known issue - future update to end on a roundabout
```

Comment:
What instruction is announced when you end on a roundabout now?

Reply:
It has the ordinal exit value included